src/backend/TrafficCourts/Common/OpenAPIs/OracleDataAPI/v1_0/OracleDataApi.v1_0.json

```diff
@@ -30,16 +30,16 @@
           "required": true
         },
         "responses": {
-          "400": {
-            "description": "Bad Request.",
+          "404": {
+            "description": "JJDispute record not found. Update failed.",
             "content": { "*/*": { "schema": { "type": "object" } } }
           },
           "405": {
             "description": "An invalid JJ Dispute status is provided. Update failed.",
             "content": { "*/*": { "schema": { "type": "object" } } }
           },
-          "404": {
-            "description": "JJDispute record not found. Update failed.",
+          "400": {
+            "description": "Bad Request.",

```

Comment:
Is there something in particular you changed in the OracleDataApi OpenAPI spec? As far as I can tell nothing has changed, except the usual random ordering of the responses.  Just wondering why this change in included in the PR.

Reply:
no changes here except i generally re-import the JSON  and rebuild to see if any changes